samples/mesh/main.c

```diff
@@ -297,20 +297,17 @@ static void init_shader(void)
     p = pb_begin();
 
     /* Set run address of shader */
-    pb_push(p++, NV097_SET_TRANSFORM_PROGRAM_START, 1); *(p++)=0;
+    p = pb_push1(p, NV097_SET_TRANSFORM_PROGRAM_START, 0);
 
     /* Set execution mode */
-    pb_push1(p, NV097_SET_TRANSFORM_EXECUTION_MODE,
-        MASK(NV097_SET_TRANSFORM_EXECUTION_MODE_MODE, NV097_SET_TRANSFORM_EXECUTION_MODE_MODE_PROGRAM)
-        | MASK(NV097_SET_TRANSFORM_EXECUTION_MODE_RANGE_MODE, NV097_SET_TRANSFORM_EXECUTION_MODE_RANGE_MODE_PRIV));
-    p += 2;
+    p = pb_push1(p, NV097_SET_TRANSFORM_EXECUTION_MODE,
+                 MASK(NV097_SET_TRANSFORM_EXECUTION_MODE_MODE, NV097_SET_TRANSFORM_EXECUTION_MODE_MODE_PROGRAM)
+                 | MASK(NV097_SET_TRANSFORM_EXECUTION_MODE_RANGE_MODE, NV097_SET_TRANSFORM_EXECUTION_MODE_RANGE_MODE_PRIV));
 
-    pb_push1(p, NV097_SET_TRANSFORM_PROGRAM_CXT_WRITE_EN, 0);
-    p += 2;
+    p = pb_push1(p, NV097_SET_TRANSFORM_PROGRAM_CXT_WRITE_EN, 0);
 
     /* Set cursor and begin copying program */
-    pb_push1(p, NV097_SET_TRANSFORM_PROGRAM_LOAD, 0);
-    p += 2;
+    p = pb_push1(p, NV097_SET_TRANSFORM_PROGRAM_LOAD, 0);
```

Comment:
This change conflicts with #187 which will have to be merged first (as this PR would hit `assert` otherwise).

The exact same code is present in the triangle sample. I say, fix it here, so mesh sample itself is consistently using new API; and we can fix it in triangle in a follow-up PR.

Reply:
triangle sample is also updated by this PR now, so this comment isn't relevant anymore.